static/js/i18n/strings/en/units.json

```diff
@@ -34,5 +34,11 @@
   },
   "celsius-display": {
     "defaultMessage": "°c"
+  },
+  "micro-gram-per-cubic-meter": {
+    "defaultMessage": "{0} μg/m3"
```

Comment:
should we use the superscript character for this? µg/m³

Reply:
Done